L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -424,6 +426,7 @@
         nEMCALO_EGIN = 5,  # to be defined
         nEM_EGOUT = 5,        # to be defined
         doBremRecovery=True,
+        doEndcapHwQual=True,
```

Comment:
Is it intended to apply also to HF> 
(maybe it doesn't matter since I don't think we're running any e/gamma there)